pyogrio/_io.pyx

```diff
@@ -1523,23 +1523,26 @@ def ogr_read_info(
         fields = get_fields(ogr_layer, encoding)
 
         meta = {
-            'crs': get_crs(ogr_layer),
-            'encoding': encoding,
-            'fields': fields[:,2], # return only names
-            'dtypes': fields[:,3],
-            'geometry_type': get_geometry_type(ogr_layer),
-            'features': get_feature_count(ogr_layer, force_feature_count),
-            'total_bounds': get_total_bounds(ogr_layer, force_total_bounds),
-            'driver': get_driver(ogr_dataset),
+            "name": get_string(OGR_L_GetName(ogr_layer)),
+            "crs": get_crs(ogr_layer),
+            "encoding": encoding,
+            "fields": fields[:,2], # return only names
+            "dtypes": fields[:,3],
+            "fid_column": get_string(OGR_L_GetFIDColumn(ogr_layer)),
+            "geometry_name": get_string(OGR_L_GetGeometryColumn(ogr_layer)),
```

Comment:
```suggestion
            "geometry_field": get_string(OGR_L_GetGeometryColumn(ogr_layer)),
```

Reply:
We already use `geometry_name` in some of the Arrow code paths (`open_arrow` uses that key in the meta dict it returns). Not necessarily a reason to change it here, but it would give a slight inconsistency.

We should maybe also think about what this key should be if there are multiple geometry fields.

Reply:
Ah yes, fine to leave it as `geometry_name` then (for now).

To handle multiple geometries, would we want this to be a list instead? (and maybe also be a plural, e.g., `geometry_names`: [...])

So `["geometry"]` for single-geometry drivers and datasets with only geometry, and `["geom", "geom2", ...]` for multiple geometries?  Then update the Arrow implementation to use the first by default?

(still haven't gotten my head fully around multiple geometry datasets)

Reply:
Typically (sometimes) there is also a "primary" geometry column I think...